src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java

```diff
@@ -70,4 +78,13 @@ private String getCaseId(Object dto) {
       caseId = uuid.toString();
     return caseId;
   }
+
+  public Message convertJSONToMessage(String messageJSON) {
+    Message gatewayMessage;
+    MessageProperties messageProperties = new MessageProperties();
+    messageProperties.setContentType("application/json");
+    MessageConverter messageConverter = new Jackson2JsonMessageConverter();
+
+    return gatewayMessage = messageConverter.toMessage(messageJSON, messageProperties);
```

Comment:
maybe just get rid 
Message gatewayMessage;

and return
return  messageConverter.toMessage(messageJSON, messageProperties);

Reply:
Have updated the code.